pkg/identity/service.go

```
@@ -142,6 +152,18 @@ func Register(g *grpc.Server, s *Service) {
 	pb.RegisterIdentityServer(g, s)
 }
 
+// ensureIssuerStillValid should check that the CA is still good time wise
+// and verifies just fine with the provided trust anchors
+func (svc *Service) ensureIssuerStillValid() error {
+	issuer := *svc.issuer
+	switch is := issuer.(type) {
+	case *tls.CA:
+		return is.Cred.Verify(svc.trustAnchorsPool, svc.expectedName)
+	default:
+		return nil
```

I think we should return an error here.